README.md

```diff
@@ -44,30 +44,89 @@ cd /aosp/src && /aosp/src/tools/base/bazel/bazel build //tools/adt/idea/android:
 
 ![image.png](images/kEpwT0nHhNX.png)
 
-Now go back to the host, open `$AOSP_ROOT/tools/adt/idea` in IDEA and click `rebuild project`
+4. Now go back to the host, open `$AOSP_ROOT/tools/adt/idea` in IDEA and click `rebuild project`
+* Make sure to set the `SDK_PLATFORM` Path Variable to `linux/android-studio` (https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-main:studio/README.md)
 * Use `Android Studio` run configuration to run Android Studio from sources.
+   * Android Studio will be launched using the distribution provided in the prebuilts/ folder.
+
+# Step-by-step guide (for Mac)
```

Comment:
Is it true that we don't need any docker for Mac M1? If so, this probably mean that google has fixed missing build tools, and we don't need docker anymore for Linux either.

Reply:
Yes.